spec.md

```diff
@@ -345,19 +346,19 @@ When the broker receives a provision request from the platform, it MUST take wha
 The `:instance_id` of a service instance is provided by the platform. This ID will be used for future requests (bind and deprovision), so the broker will use it to correlate the resource it creates.
 
 #### Parameters ####
-| Parameter name  | Type  |  Description |
+| Parameter name | Type | Description |
 |---|---|---|
-|  accepts_incomplete | boolean  | A value of true indicates that the marketplace and its clients support asynchronous broker operations. If this parameter is not included in the request, and the broker can only provision an instance of the requested plan asynchronously, the broker should reject the request with a 422 as described below.  |
+| accepts_incomplete | boolean | A value of true indicates that the marketplace and its clients support asynchronous broker operations. If this parameter is not included in the request, and the broker can only provision an instance of the requested plan asynchronously, the broker should reject the request with a 422 as described below. |
```

Comment:
Should this be "the broker should reject the request with a `422 Unprocessable Entity` as described below" for consistency?